docs/introduction.md

```diff
@@ -427,19 +128,12 @@ You can then copy part of this data folder to be used in DPS:
 * `trie/execution/NodeID` can be given as `trie`
 * `data/consensus/NodeID/checkpoint.00000001` can be given as `root.checkpoint`
 
-You can then run the `flow-dps-indexer`, and it should properly build its index based on the given information.
+You can then run the `flow-dps-indexer`, which should properly build its index based on the given information.
 Be careful, there should never be multiple indexers running in parallel sharing the same `index` folder as they might end up corrupting the contents of the database.
```

Comment:
I think this is unnecessary, badger won't allow it anyway, due to a file system lock for each DB.